agent/plugins/logging/base/src/main/java/org/apache/shardingsphere/agent/plugin/logging/base/advice/MetaDataContextsFactoryAdvice.java

```diff
@@ -25,7 +25,7 @@
 import java.lang.reflect.Method;
 
 /**
- * Schema meta data loader advice.
+ * Schema metadata loader advice.
  */
 @Slf4j
 public final class MetaDataContextsFactoryAdvice implements ClassStaticMethodAroundAdvice {
```

Comment:
Hi @DivvSaxena 
Why did you change the comment but without the class name?

Reply:
Hi @RaigorJiang  It was intially "meta data" i just changed it into "metadata".

Reply:
Hi, I mean the class name `MetaDataContextsFactoryAdvice` should also be changed to `MetadataContextsFactoryAdvice`.   
`MetaData` -> `Metadata`

Reply:
Hey @RaigorJiang  , Ok Updating.